src/cloud/cdn/cloud-vcl-custom-snippets.md

```diff
@@ -43,7 +43,7 @@ The examples and instructions in the {{ site.data.var.ece }} documentation expla
 
 You can create and manage custom VCL snippets from the Magento Admin UI or by using the Fastly API:
 
--  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI.
+-  [Magento Admin UI](#manage-custom-vcl-snippets-from-the-magento-admin-ui)—We recommend using the Magento Admin UI to manage custom VCL snippets because it automates the process to validate and upload the custom snippet and apply your changes to the Fastly service configuration. Additionally, you can view and edit the custom VCL snippets added to the Fastly service configuration from the Admin UI. **Please note:** deleting a snippet from the Magento UI only removes it from the Magento UI, to fully delete requires this to be done via the Fastly API. As Cloud projects do not get access to the Fastly UI, it is recommended that an alternative to deleting via the Fastly UI is to set the VCL contents to empty content and upload the VCL via the Magento Admin UI.
```

Comment:
This is not true. If a customer has an API key, they can delete the snippet themselves using an API call.  

The API call to delete a snippet is documented in the Quick reference section here: https://devdocs.magento.com/cloud/cdn/cloud-vcl-custom-snippets.html#manage-vcl

It is also documented in the Delete Custom snippet section in the individual custom VCL example articles: https://devdocs.magento.com/cloud/cdn/fastly-vcl-badreferer.html#delete-the-custom-vcl-snippet